src/backend/aspen/workflows/nextstrain_run/run_nextstrain_ondemand.sh

```diff
@@ -52,16 +60,17 @@ aligned_gisaid_s3_bucket=$(echo "${aligned_gisaid_location}" | jq -r .bucket)
 aligned_gisaid_sequences_s3_key=$(echo "${aligned_gisaid_location}" | jq -r .sequences_key)
 aligned_gisaid_metadata_s3_key=$(echo "${aligned_gisaid_location}" | jq -r .metadata_key)
 
+
 # fetch the gisaid dataset
-aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_sequences_s3_key}" /ncov/results/
-aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_metadata_s3_key}" /ncov/results/
+$aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_sequences_s3_key}" /ncov/results/
+$aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_metadata_s3_key}" /ncov/results/
 
 # run snakemake, if run fails export the logs from snakemake and ncov to s3 
-(cd /ncov && snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/ --resources=mem_mb=312320) || { aws s3 cp /ncov/.snakemake/log/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_date}/${S3_FILESTEM}/${WORKFLOW_ID}/logs/snakemake/" --recursive ; aws s3 cp /ncov/logs/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_date}/${S3_FILESTEM}/${WORKFLOW_ID}/logs/ncov/" --recursive ; }
+(cd /ncov && snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/ --resources=mem_mb=312320) || { $aws s3 cp /ncov/.snakemake/log/ "${s3_prefix}/logs/snakemake/" --recursive ; $aws s3 cp /ncov/logs/ "${s3_prefix}/logs/ncov/" --recursive ; }
 
 # upload the tree to S3
-key="phylo_run/${build_date}/${S3_FILESTEM}/${WORKFLOW_ID}/ncov.json"
-aws s3 cp /ncov/auspice/ncov_aspen.json "s3://${aspen_s3_db_bucket}/${key}"
+key="${key_prefix}/ncov_aspen.json"
+$aws s3 cp /ncov/auspice/ncov_aspen.json "s3://${aspen_s3_db_bucket}/${key}"
```

Comment:
sometimes we get > 1 JSON in 1 folder, so in the older code JSONs are renamed in [this step] (the builds.yaml needs too). Is the current set up, each tree run will have its own folder and this won't happen anymore? (https://github.com/chanzuckerberg/aspen/blob/e87501146dcf9106e9c39e309fb848b584f48166/src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh#L72)
![image](https://user-images.githubusercontent.com/20667188/146257996-44c9b5a5-e695-468a-9ca0-2739ade51bf9.png)

Reply:
> sometimes we get > 1 JSON in 1 folder, so in the older code JSONs are renamed in [this step](https://github.com/chanzuckerberg/aspen/blob/e87501146dcf9106e9c39e309fb848b584f48166/src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh#L72) (the builds.yaml needs too). Is the current set up, each tree run will have its own folder and this won't happen anymore? ![image](https://user-images.githubusercontent.com/20667188/146257996-44c9b5a5-e695-468a-9ca0-2739ade51bf9.png)

The new path will create a folder prefix for every phylo tree:
https://github.com/chanzuckerberg/aspen/pull/868/files#diff-510b082ad0018f47841b06cb3ebcc9910c1e3487976975fcb68ff759f89b0dbbR29

So the files will be like `/phylo_run/Tuolomne Contextual/12345/ncov_aspen.json`

Reply:
i'll add this to my tree wiki!

Reply:
We can also change the way we format these paths -- if there's anything that you think is easier to understand, let me know!

Reply:
this is going to make the tree debug a lot easier. i spend a lot of time clicking through the 1234 folders every time TnT